WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -311,10 +344,30 @@ private extension CommentDetailViewController {
         navigationItem.standardAppearance = isBlurred ? blurredBarAppearance : opaqueBarAppearance
     }
 
-    func configureEditButtonItem() {
-        navigationItem.rightBarButtonItem = comment.allowsModeration() ? UIBarButtonItem(barButtonSystemItem: .edit,
-                                                                                         target: self,
-                                                                                         action: #selector(editButtonTapped)) : nil
+    func configureNavBarButtons() {
+        var barButtonItems: [UIBarButtonItem] = []
+
+        if isNotificationComment && splitViewControllerIsHorizontallyCompact {
+            barButtonItems.append(navigationButtons())
+        }
+
+        if comment.allowsModeration() {
+            barButtonItems.append(editBarButtonItem)
+        }
+
+        navigationItem.setRightBarButtonItems(barButtonItems, animated: false)
+    }
+
+    func navigationButtons() -> UIBarButtonItem {
```

Comment:
Nitpick, but I'd either rename this to `makeNavigationButtons` or move it to a lazy var named something like `navigationButtonsStackView`.